circus-train-core/src/test/resources/log4j.xml

```diff
@@ -24,8 +24,7 @@
   </appender>
 
   <logger name="com.hotels.bdp.circustrain">
-    <!-- Required to be DEBUG for com.hotels.bdp.circustrain.core.metastore.HiveConfFactoryTest.secretProperties() -->
-    <level value="DEBUG" />
+    <level value="INFO" />
```

Comment:
We should have had two separate PRs - one for getting the build working in Travis and the other for the partition table changes. Up to you whether to leave like this for now or split.

Reply:
This fix is not completely related to travis, I've made the test more robust

Reply:
Yeah, I meant in general with the various other changes to logging config etc.